recipes/pnglatex/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
This file is already included in the distribution so you can just delete it here.